File: Swift/012_Program/main.swift
import Foundation

print("=== Swift Program 012 ===")
print("Demonstrating basic Swift features")

// Variables and constants
let constant = "Cannot be changed"
var variable = "Can be changed"

print("Constant: \(constant)")
print("Variable: \(variable)")

// String interpolation
let programNumber = 12
print("This is program number \(programNumber)")

// Control flow
switch programNumber {
case 1...10:
    print("Single digit")
case 11...99:
    print("Double digit")
default:
    print("Three or more digits")
}

File: Kotlin/012_Program/Main.kt
fun main() {
    println("=== Kotlin Program 012 ===")
    println("Demonstrating basic Kotlin features")

    // Variables
    val immutable = "Cannot be changed"
    var mutable = "Can be changed"

    // String interpolation
    println("Immutable: $immutable")
    println("Mutable: $mutable")

    // When expression
    val result = when (val x = 12) {
        in 1..10 -> "Single digit"
        in 11..99 -> "Double digit"
        else -> "Three or more digits"
    }
    println("Result: $result")
}
